test/lib/config-env-flags.js

```diff
@@ -41,16 +41,22 @@ describe('lib: config/env-flags', function() {
       flags: {
         silent: false,
         gulpfile: '/path/to/gulpfile',
+        require: ['a', 'b'],
       },
     };
 
-    var result =  mergeConfig(env, config);
+    var opts = {
+      gulpfile: env.configPath,
+      require: 'preloat',
```

Comment:
Is this a typo?

Reply:
I'm going to remove this line because we don't actually use the `opts` like this.  We thread the `require` property through the `env` produced by `liftoff.prepare`